docs/book/src/plugins/plugins-versioning.md

```diff
@@ -0,0 +1,52 @@
+# Plugins Versioning
```

Comment:
See that one of the ci jobs previews these changes in the docs: 

<img width="844" alt="Screenshot 2022-04-02 at 09 51 31" src="https://user-images.githubusercontent.com/7708031/161375516-d420870a-42d7-4124-aa2d-d07be58442d4.png">

You can check your PR changed in : https://deploy-preview-2588--kubebuilder.netlify.app/plugins/plugins-versioning.html

Reply:
So that, you can see that the markdown table is not rendering fine: 

<img width="886" alt="Screenshot 2022-04-02 at 09 52 23" src="https://user-images.githubusercontent.com/7708031/161375569-723bf827-8571-4a8d-8429-1e5cae724fcb.png">

You can check in the Migration Guide from v2 to V3 an example over how to create a table, see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/migration/manually_migration_guide_v2_v3.md?plain=1#L133-L157

Reply:
Okay. I will add the table properly